cdk/config.ts

```diff
@@ -18,6 +19,8 @@ export class Config {
     this.jwksUrl = process.env.JWKS_URL;
     if (!process.env.DATA_ACCESS_ROLE_ARN) throw Error("Must provide DATA_ACCESS_ROLE_ARN");
     this.dataAccessRoleArn = process.env.DATA_ACCESS_ROLE_ARN!;
+    if (!process.env.STAC_API_INTEGRATION_API_ARN) throw Error("Must provide STAC_API_INTEGRATION_API_ARN");
```

Comment:
Aren't you _forcing_ the user to provide an API ARN in this case? The description would make it seem like it would be optional as in a default would be set

Reply:
It meant to have it required, but now that you say that I don't think it should be. Like above, addressing that in a follow up PR.